test/y2partitioner/widgets/configurable_blk_devices_table_test.rb

```diff
@@ -33,6 +33,39 @@
     end
   end
 
+  describe "#init" do
+    context "UIState contain sid of device in table" do
+      it "sets value to row with the device" do
+        valid_sid = devices.last.sid
+        Y2Partitioner::UIState.instance.select_row(valid_sid)
+
+        expect(subject).to receive(:value=).with(subject.send(:row_id, valid_sid))
+
+        subject.init
+      end
+    end
+
+    context "UIState contain sid that is not in table" do
+      it "selects any valid device in table" do
+        Y2Partitioner::UIState.instance.select_row("999999999")
+
+        expect(subject).to receive(:value=)
```

Comment:
I would check here that the argument is different from `999999999` and is one of `devices.map(&:sid)`

Reply:
makes sense